repctl/pkg/cmd/snapshot.go

```diff
@@ -0,0 +1,167 @@
+/*
+ Copyright © 2023 Dell Inc. or its subsidiaries. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/dell/repctl/pkg/config"
+	"github.com/dell/repctl/pkg/k8s"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+// GetSnapshotCommand returns 'snapshot' cobra command
+/* #nosec G104 */
+func GetSnapshotCommand() *cobra.Command {
+	snapshotCmd := &cobra.Command{
+		Use:   "snapshot",
+		Short: "allows to execute snapshot action at the specified cluster or rg",
+		Example: `
+For single cluster config:
+./repctl --rg <rg-id> --sn-namespace <namespace> --sn-class <snapshot class> snapshot`,
+		Long: `
+This command will perform a snapshot at specified cluster or at the RG.\n
+Note: More information to be added.`,
```

Comment:
Is this a note to you to add additional explanation?

Reply:
I will address this in an upcoming PR.